src/field_theory/is_alg_closed/basic.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kenny Lau
 -/
 
+import field_theory.normal
```

Comment:
The big question: should we import from the file that defines the property to a definition that has the property, or vice versa? I think this is the right direction in this case, `normal` has pretty light imports and `is_alg_closed` quite heavy.